CHANGELOG.md

```diff
@@ -7,6 +7,13 @@
   - `-<prefix>.s3.sse.type`
   - `-<prefix>.s3.sse.kms-key-id`
   - `-<prefix>.s3.sse.kms-encryption-context`
+* [ENHANCEMENT] Ruler: Add TLS and explicit basis authentication configuration options for the HTTP client the ruler uses to communicate with the alertmanager. #3752
```

Comment:
Please remember to update the CLI flags mentioned below accordingly, once you will change them in the code 🙏

Reply:
👍